.travis.yml

```diff
@@ -66,38 +69,35 @@ matrix:
     - env: MODULE='cql' ARGS='-Dtest=**/graphdb/cql/* -Dtest.skip.murmur=true'
     - env: MODULE='cql' ARGS='-Dtest=**/graphdb/cql/* -Dtest.skip.byteorderedpartitioner=true -Dtest.skip.murmur-serial=true -Dtest.skip.murmur-ssl=true'
 
-addons:
-  coverity_scan:
-    # Coverity config parameters described in detail:
-    # https://scan.coverity.com/travis_ci
-    project:
-      name: "JanusGraph/janusgraph"
-      version: "0.2.0-SNAPSHOT"
-      description: "Scalable, distributed graph database"
-    notification_email: janusgraph-ci@googlegroups.com
-    build_command_prepend:
-      - if ! [ -v COVERITY_ONLY ]; then
-          echo "Skipping Coverity for non-Coverity job";
-          exit 0;
-        fi
-    build_command: mvn clean package -DskipTests=true --batch-mode
-    branch_pattern: coverity_scan
-
 install:
-  # Build and install current module and dependencies. Handle output timeouts and retry on error.
-  - if [ "${COVERITY_SCAN_BRANCH}" == 1 ] || [ -v COVERITY_ONLY ]; then
-      echo "Skipping install build step in Coverity branch/job";
+  - if [ "${TRAVIS_BRANCH}" == "${COVERITY_BRANCH_NAME}" ] && ! [ -z ${COVERITY_ONLY:-} ]; then
```

Comment:
I may be paranoid, but I typically encode variables in double-quotes, i.e., ` [ -z "${COVERITY_ONLY:-}" ]`, here and below.